kingfisher_scrapy/spiders/nigeria_budeshi_releases.py

```diff
@@ -21,3 +21,14 @@ def build_urls(self, project):
         url = 'https://budeshi.ng/api/releases/{id}/{tag}'
         for tag in ('planning', 'tender', 'award', 'contract'):
             yield self.build_request(url.format(id=project['id'], tag=tag), formatter=components(-2))
+
+    def parse(self, response):
+        data = response.json()
+        # some responses include a release list with null objects, eg:
+        #   "releases": [
+        #     null
+        #   ]
+        if 'releases' in data and data['releases'] and any(release is None for release in data['releases']):
+            yield self.build_file_error_from_response(response, errors=data)
+        else:
+            yield from super().parse(response)
```

Comment:
Do we want to reject the data entirely, or do:

```suggestion
        if 'releases' in data:
            data['releases'] = filter(None, data['releases'])
            yield self.build_file_from_response(response, data_type=self.data_type, data=data)
```

Reply:
Also, do we need the `if`? If all responses are expected to set it, then we do not need it.

Reply:
Actually, all the cases that I've seen are with just one null object, and if we do the `filter` we will end up with empty packages, don't we?

Reply:
Let's simplify the `if` then - the current code suggests that some entries in the list might not be null.

Reply:
> Let's simplify the if then to just if not data['releases']

That will not work as the list is `[None]`, I will change it to `data['releases'][0] is None`

Reply:
Yes - I realized before going to asleep, lol. Even clearer is `data['releases'] == [None]`